crates/rspack/tests/tree-shaking/named-export-decl-with-src-eval/snapshot/snap.diff

```diff
@@ -0,0 +1,11 @@
+--- expected
++++ actual
+@@ -41,7 +41,7 @@
+ var L = _Layout__WEBPACK_IMPORTED_MODULE_0__["default"];
+ L.something = _Something__WEBPACK_IMPORTED_MODULE_1__.something;
+ 
+- var LL = L;
```

Comment:
align to webpack : 
![image](https://github.com/web-infra-dev/rspack/assets/17974631/5ad047e8-532b-4e12-a769-cc9e5d130916)